CesiumGS/cesium
# Update sandcastle examples to 3DTiles 1.1

# Description



Update the Sandcastle examples under 3DTiles Next to 3DTiles 1.1 including the assets in ion. Also completely removes references to 3DTiles Next in Sandcastle





## Issue number and link

Fixes https://github.com/CesiumGS/cesium/issues/11668




## Testing plan

- Load Sandcastle locally (`npm start`)
- Load each of the examples below in the 3DTiles category and verify they work as expected
  - 3D Tiles 1.1 CDB Yemen
  - 3D Tiles 1.1 Photogrammetry
  - 3D Tiles 1.1 Photogrammetry Classification
  - 3D Tiles 1.1 S2 Globe
  - Custom Shaders 3D Tiles
      - This one uses an Asset Depot asset which will need to be updated as part of a separate effort
  - Custom Shaders Models
  - Custom Shaders Property Textures



# Author checklist

- [ ] I have added or updated unit tests to ensure consistant code coverage
